csharp/ql/src/semmle/code/csharp/controlflow/ControlFlowGraph.qll

```diff
@@ -1576,25 +1575,22 @@ module ControlFlow {
       }
 
       pragma [nomagic]
-      private ControlFlowElement lastConstCaseStmt(ConstCase cc, Completion c) {
-        result = last(cc.getStmt(), c)
+      private ControlFlowElement lastCaseStmt(CaseStmt cs, Completion c) {
+        result = last(cs.(TypeCase).getStmt(), c)
+        or
+        result = last(cs.(ConstCase).getStmt(), c)
```

Comment:
Could replace with
```
result = last(cs.(LabeledStmt).getStmt(), c)
```
Feel free to change if you like.

Reply:
I did that initially but additional edges appeared.